.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,65 @@
+name: Continuous integration
+on: [push]
+
+jobs:
+  test-macos-nixpkgs:
+    name: Build & Test - macOS (Nixpkgs)
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cachix/install-nix-action@v12
+        with:
+          nix_path: nixpkgs=./nixpkgs/default.nix
```

Comment:
IIUC this no longer enables Cachix caching itself. Looking at the docs it seems reasonably straight forward, assuming that the secrets are available to GH actions: https://nix.dev/tutorials/continuous-integration-github-actions.html.

Reply:
I'm not sure Cachix is necessary, since the official binary cache has a 100% cache hit rate for all the derivations in this PR.

Reply:
Sounds good, we can still add it later if we need custom overrides at some point.